examples/bow.py

```diff
@@ -16,13 +16,13 @@
 import torch
 
 # Set PATHs
-PATH_TO_SENTEVAL = '../senteval'
+PATH_TO_SENTEVAL = '../'
 PATH_TO_DATA = '../data/senteval_data'
 PATH_TO_GLOVE = 'glove/glove.840B.300d.txt'
 
 # import SentEval
 sys.path.insert(0, PATH_TO_SENTEVAL)
-import senteval
+from senteval.senteval import SentEval
```

Comment:
Is there a way we could avoid this repetition by putting the "senteval.py" in the root directory of the SentEval package? (instead of in senteval/)?

That would be nicer to just have "import senteval" and then "senteval.SentEval".